README.md

```diff
@@ -180,4 +180,4 @@ This software is released under the MIT license. See [the license file](LICENSE)
 [coveralls-badge]: https://coveralls.io/repos/juanjoDiaz/serverless-middleware/badge.svg?branch=master
 [coveralls-badge-url]: https://coveralls.io/r/juanjoDiaz/serverless-middleware?branch=master
 [dev-badge]: https://david-dm.org/juanjoDiaz/serverless-middleware.svg
-[dev-badge-url]: https://david-dm.org/juanjoDiaz/serverless-middleware
\ No newline at end of file
+[dev-badge-url]: https://david-dm.org/juanjoDiaz/serverless-middleware
```

Comment:
This line change shouldn't be here. Probably you have accidentally removed the empty line at the end.

Reply:
Weird, I just used GitHub's online editor. I'll fix it.

Reply:
Strange, my editor shows the newline in there. You can close this PR if you want, and maybe just update the spelling in your own branch.